packages/Interface/src/DTK_UserApplication.hpp

```diff
@@ -77,12 +77,15 @@ class UserApplication
     PolyhedronList<Kokkos::LayoutLeft, ExecutionSpace> getPolyhedronList();
 
     //! Get a cell list from the application.
-    CellList<Kokkos::LayoutLeft, ExecutionSpace>
-    getCellList( std::vector<std::string> &cell_topologies );
+    CellList<Kokkos::LayoutLeft, ExecutionSpace> getCellList();
 
     //! Get a boundary from the application and put it in the given list.
     template <class ListType>
-    void getBoundary( const std::string &boundary_name, ListType &list );
+    void getBoundary( ListType &list );
```

Comment:
A map will only work for a single boundary. This fixes that problem and also doesn't force the user to identify boundaries with a string. Whichever boundary the user wants to give to the map they can specify in whichever way they want with the user data they register with the function.